spring-ai-core/src/main/java/org/springframework/ai/aot/AiRuntimeHints.java

```diff
@@ -40,6 +40,10 @@
  */
 public class AiRuntimeHints {
 
+	private AiRuntimeHints() {
```

Comment:
Similar to before, will create as abstract.  Likely should change the name to RuntimeHintUtils, but that will be a big commit, so will do another time.